weneige/orders/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from core.create_datetime import BaseModel
+from users.models          import User
+from products.models       import ProductOption
+
+class Order(BaseModel):
+    user           = models.ForeignKey(User, on_delete=models.CASCADE, related_name="user_order")
+    product_option = models.ForeignKey(ProductOption, on_delete=models.CASCADE, related_name="option_order")
+    quantity       = models.IntegerField()
+    address        = models.CharField(max_length=100)
+    mobile_number  = models.CharField(max_length=100)
+
```

Comment:
```suggestion
class Order(BaseModel):
    user           = models.ForeignKey(User, on_delete=models.CASCADE, related_name="user_order")
    product_option = models.ForeignKey(ProductOption, on_delete=models.CASCADE, related_name="option_order")
    quantity       = models.IntegerField()
    address        = models.CharField(max_length=100)
    mobile_number  = models.CharField(max_length=100)
    order_number = models.CharField(max_length=30)
    order_status = models.ForewignKey(OrderStatus, on_delete=models.CASCADE)
```
order_number 컬럼 추가 및 OrderStatus 테이블도 작성하여 관계를 추가해주세요~~